apps/site/next.dynamic.mjs

```diff
@@ -110,9 +112,21 @@ const getDynamicRouter = async () => {
 
     // This verifies if the given pathname actually exists on our Map
     // meaning that the route exists on the website and can be rendered
-    if (pathnameToFilename.has(normalizedPathname)) {
-      const filename = pathnameToFilename.get(normalizedPathname);
-      const filepath = join(process.cwd(), 'pages', locale, filename);
+    if (
```

Comment:
TODO for myself: by debugging this I was that parts of the filepath were missing that's why the changes here were applied, I need to double check this and see why we need to do that now... maybe something somewhere else got changed incorrectly?

Reply:
outdated/fixed 😄